ddl/placement/bundle.go

```diff
@@ -217,6 +205,10 @@ func newBundleFromOptions(options *model.PlacementSettings) (bundle *Bundle, err
 		isSyntaxSugar = false
 	}
 
+	if options.Followers == 0 {
```

Comment:
I found a new problem... `create table t1(id int) followers=0` still create 2 followers...

Reply:
But anyway, we can handle it later to add a variable `IsFollowerSet` in PlacementSettings ...

Reply:
It is forbidden for now. Let us leave the problem for future.